keyboards/edi/hardlight/hardlight.h

```diff
@@ -0,0 +1,43 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef HARDLIGHT_H
+#define HARDLIGHT_H
+
+#include "quantum.h"
+#define ___ KC_NO
```

Comment:
```suggestion
```

`___`/`KC_NO` is unused in the LAYOUT macro, and further `___` could be easily confused with the core define `_______` which is aliased to `KC_TRNS`.  Best to avoid altogether.